src/platform_impl/macos/view.rs

```diff
@@ -637,7 +699,7 @@ fn update_potentially_stale_modifiers(state: &mut ViewState, event: id) {
     }
 }
 
-extern "C" fn key_down(this: &Object, _sel: Sel, event: id) {
+extern "C" fn key_down(this: &mut Object, _sel: Sel, event: id) {
```

Comment:
The `this` parameter doesn't have to be a `mut` anymore.

Reply:
Thanks !!